### PR TITLE
[RFR] 🇷🇺 heading hyphenation

### PR DESCRIPTION
**Pull request by Grawl**

### CODE REVIEW

File: _sass/base/_typography.scss [hunk 1]
```diff
@@ -61,11 +61,17 @@ h3 {
...
-// See https://github.com/HugoGiraudel/sass-guidelines/issues/163
 :lang(ru) {
+  // Russian l10n extensions
   h1, h2, h3 {
+    // Do not capitalize every word in a header
+    // See https://github.com/HugoGiraudel/sass-guidelines/issues/163
     text-transform: initial;
+    // Use hyphens
+    // See https://github.com/HugoGiraudel/sass-guidelines/issues/166#issuecomment-91924154
+    -webkit-hyphens: auto;
```
Comment (Grawl) [inline _sass/base/_typography.scss]: it's not handy to write prefixes manually, maybe you will use Autoprefixer?

**Comment by HugoGiraudel** [comment]
I think the `hyphens` thingie is not restricted to Russian and could be applied to all languages. What do you think?

**Comment by HugoGiraudel** [comment]
Regarding prefixing, there is the `prefix` mixin

File: _sass/base/_typography.scss (hunk 1 above)
Comment (HugoGiraudel) [inline _sass/base/_typography.scss]: Regarding prefixing, there is the `prefix` mixin.

**Comment by HugoGiraudel** [comment]
Done on `gh-pages`: https://github.com/HugoGiraudel/sass-guidelines/commit/e59e218949facbd763ff0efcb7134e89c739e2cd.

File: _sass/base/_typography.scss (hunk 1 above)
Comment (Grawl) [inline _sass/base/_typography.scss]: Oh hell… after Autoprefixer looks like hell from 2011.

**Comment by Grawl** [comment]
Agree. My bad.

**Pull request closed without merging**